react-components/src/components/NodeCacheProvider/types.ts

```diff
@@ -7,7 +7,7 @@ import { type InModel3dEdgeProperties } from '../../utilities/globalDataModels';
 
 export type Fdm3dNodeData = { fdmId: DmsUniqueIdentifier; view: Source; cadNode: Node3D };
 export type FdmCadEdge = EdgeItem<InModel3dEdgeProperties>;
-export type FdmEdgeWithNode = { edge: FdmCadEdge; node: Node3D };
+export type FdmEdgeWithNode = { edge: FdmCadEdge; node: Node3D; view: Source };
```

Comment:
I realize that `FdmEdgeWithNode` contains the same data as `Fdm3dNodeData`, just a little more. It can be a little hard to tell them apart based on the names too.. I say we combine these two into the same type. Perhaps just `Fdm3dNodeData = { edge: FdmCadEdge; node: Node3D; view: Source };`?

Reply:
Yeah, sounds good, will do that.